### DISCUSSION TITLE
Unable to import via CSV and sqlite editor

### DISCUSSION THREAD

**Discussion by WashedUpUnderwear** [stashapp/stash#3404] [Q&A] [answered]
Hi,

Discovered stashapp last week, and tried to import what I already have.
Scenes are rated in windows media player from 1-5, Tag are with playlists and studios are from folder in explorer.

What I did is using excel, DB Browser for sqlite and CSV Export/Import, I was able to edit the scenes table to add the rating in the form of 100 - 80 - 60 - 40 - 20 and to add a date as it add automaticaly 0001-01-01 in the table next to the ratign field when you saved.

Now when I relauch stashapp, I can see all the scenes but they are empty. No file name, no nothing, just a blank scene with a rating.

It looks like this
![image](https://user-images.githubusercontent.com/124624916/217066951-6c9b14e5-ea65-47c7-9b26-2ea4ad3ae102.png)

They were working fine before editing the table I tried to check if data were changing in another table that would create this but I didn't find anything.

Is there a way, using windows and whatever software to edit the table using CSV ? is UTF-8 the right encoding ?

I can't really start from nothing as it would take ages to redo every video one by one

Thanks

**Comment by DogmaDragon** [comment, marked as answer]
You can use `gql-iterate` to import via CSV through GraphQL. Guide: https://discourse.stashapp.cc/t/bulk-importing-via-csv-using-gql-iterate/4461

The schema is available here <https://github.com/stashapp/stash/tree/master/graphql>.

**Reply by WashedUpUnderwear** [reply]
Thanks for your reply.

I don't have access to a linux machine and my knowledge on linux is next to 0 but I will try

**Reply by Flashy78** [reply]
You can install it with NodeJS package manager npm, which is available in Windows as well.

**Reply by WashedUpUnderwear** [reply]
> You can install it with NodeJS package manager npm, which is available in Windows as well.

Thanks, I will try that. Didn't know it was possible. Will report back
